.circleci/config.yml

```diff
@@ -111,6 +111,7 @@ commands:
 
 jobs:
   runtest:
+    resource_class: macos.x86.medium.gen2
```

Comment:
I just checked and looks like this can't be specified for all jobs at once, so this looks good 👍